### ISSUE TITLE
question on mapping to collections

### ISSUE THREAD

**Issue by alexfrieden** [VeliovGroup/ostrio-Neo4jreactivity#28]
So we have originally used mongo and are switching to neo4jReactivity.  We have a collection bits, 

`Bits = new Meteor.Collection("bits");`

In server, we have a publications.js file to publish what is in the mongo collection.  I would like to change that to neo4j results.  

```Meteor.publish('bits', function() {
  return Bits.find();
});```

What is the best way to do that with Neo4jReactivity?  Thanks!

**Comment by dr-dimitru** [comment]
@alexfrieden as you know neo4j doesn't have any collections, so thereis nothing to publish except certain queries. For this purpose please use Meteor.neo4j.call and Meteor.neo4j.methods, they act just like publish/subscribe for mongo

**Comment by alexfrieden** [comment]
And keep those at the publication layer?

**Comment by dr-dimitru** [comment]
@alexfrieden Sorry I didn't get your question.

**Comment by alexfrieden** [comment]
@dr-dimitru let me rephrase, should the cypher queries happen in the client or the server and get pushed to the client, and how would you push it?

**Comment by dr-dimitru** [comment]
@alexfrieden You may run queries both client and server. But it has same security policy as original mongo isomorphic driver - to avoid security issues manage it via methods/call .

Once you get data via client's query() or from server via call() - retrieved data is reactive, so updates will be pushed automatically

**Comment by lifeinchords** [comment]
Thanks for the info. We're working towards moving our app to neo4J.
How does the fact that we're moving away from Meteor collection affect this: https://github.com/msavin/Mongol/blob/master/documentation/COMPATIBILITY.md

Do you think there's a way to get the best of both - neo4j, yet still retain ability to debug the neo4j "collections"?

**Comment by dr-dimitru** [comment]
@lifeinchords there is shouldn't be any issues in compatibility of neo4j driver with any other DB in Meteor. You can use Neo4j DB and any other simultaneously

**Comment by alexfrieden** [comment]
Great this helps a lot!
